week01/teach/ArraySelector.cs

```diff
@@ -7,21 +7,10 @@ public static void Run()
         var select = new[] { 1, 1, 1, 2, 2, 1, 2, 2, 2, 1};
         var intResult = ListSelector(l1, l2, select);
         Console.WriteLine("<int[]>{" + string.Join(", ", intResult) + "}"); // <int[]>{1, 2, 3, 2, 4, 4, 6, 8, 10, 5}
-
-        var l3 = new[] { 'A', 'A', 'A', 'A', 'A'};
-        var l4 = new[] { 'B', 'B', 'B', 'B', 'B'};
-        select = new[] { 1, 2, 1, 2, 1, 2, 1, 2, 1, 2};
-        var charResult = ListSelector(l3, l4, select);
-        Console.WriteLine("<char[]>{" + string.Join(", ", charResult) + "}"); // <char[]>{A, B, A, B, A, B, A, B, A, B}
     }
 
     private static int[] ListSelector(int[] list1, int[] list2, int[] select)
     {
         return new int[0];
     }
```

Comment:
The instructions tell students to implement this one. The other one serves no purpose.